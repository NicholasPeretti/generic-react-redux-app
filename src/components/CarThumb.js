import React from 'react'
import PropTypes from 'prop-types'

const style = {
  width: '40px',
  height: '30px'
}

var CarThumb = ({ url }) => (
  <img src={url} style={style} />
)

CarThumb.propTypes = {
  url: PropTypes.string.isRequired
}

export default CarThumb
