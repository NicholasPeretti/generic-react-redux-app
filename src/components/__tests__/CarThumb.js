import React from 'react'
import { shallow } from 'enzyme'
import CarThumb from '../CarThumb'
import PresentationalTest from '../../utils/tests/presentational'

const props = {
  url: '/pic/pic1.png'
}

describe('CarThumb', () => {
  PresentationalTest(CarThumb, props)

  const wrapper = shallow(<CarThumb {...props} />)

  test('Should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
