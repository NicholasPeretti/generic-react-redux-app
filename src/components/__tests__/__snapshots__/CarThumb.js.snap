// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarThumb Should match snapshot 1`] = `"<img src=\\"/pic/pic1.png\\" style=\\"width:40px;height:30px\\"/>"`;
